test/fields/collections/Relationship/e2e.spec.ts

```diff
@@ -159,11 +159,38 @@ describe('relationship', () => {
 
   test('should hide relationship add new button', async () => {
     await page.goto(url.create)
+    await page.waitForURL(url.create)
+    const count1 = await page
+      .locator('#relationWithAllowEditToFalse-add-new .relationship-add-new__add-button')
+      .count()
+    expect(count1).toEqual(1)
```

Comment:
We typically use the `.toHaveCount()` assertion which let's you skip assigning the count variable. The main thing you lose here is that this isn't retryable. https://playwright.dev/docs/test-assertions#auto-retrying-assertions